### DISCUSSION TITLE
How can we handle 2 millions plus subscriber?

### DISCUSSION THREAD

**Discussion by rajcse2005** [moscajs/aedes#798] [Q&A]
Hi any one can help me out how to handle 2 Millions plus subscriber in broker.
If anyone has example code for broker please let us know.
Guide me how to do it?

**Comment by robertsLando** [comment]
What about supporting the project first?

**Comment by Bytesware** [comment]
> What about supporting the project first?

project is about multiple subscribers connect to broker with respective topic and publisher will push message to respective subscriber when got any update.

**Reply by robertsLando** [reply]
What I mean is that handling 2M+ subscribers could be considered an "enterprise usage" and I think you should consider to pay for that kind of support

**Comment by Bytesware** [comment]
enterprise usage

how to get that support? I am ok to pay.

 

Thanks,

Rajesh

**Reply by robertsLando** [reply]
PM me at: daniel.sorridi@gmail.com

**Comment by Bytesware** [comment]
Please check.

 

Thanks,

Rajesh
